docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  mqtt2prometheus:
+    build: .
```

Comment:
Can we specifiy both a `build` and `image` in a compose file? What has precedence?